projects/fsastorefrontlib/src/core/my-account/store/reducers/index.ts

```diff
@@ -36,6 +36,7 @@ export function clearUserState(
       action.type === AuthActions.LOGOUT ||
       action.type === fromClaimAction.DELETE_CLAIM_SUCCESS
     ) {
+      localStorage.removeItem('spartacus⚿⚿claims');
```

Comment:
`spartacus⚿⚿` this is their hard-coded code in the method `generateKeyWithContext`